api/lanes.py

```diff
@@ -1304,6 +1304,13 @@ def available_facets(cls, config, facet_group_name):
         return [cls.AVAILABLE_NOW, cls.AVAILABLE_NOT_NOW,
                 cls.AVAILABLE_ALL, cls.AVAILABLE_OPEN_ACCESS]
 
+class HasSeriesFacets(Facets):
+    """A faceting object for a feed containg books guaranteed
+    to belong to _some_ series.
+    """
+    def modify_search_filter(self, filter):
```

Comment:
This might be the intended effect but the method name implies one could set this to true or false. Would direct modification of the filter.series attribute be more readable?

Reply:
We discussed this in Slack. `modify_search_filter` is the name of a hook method defined by `Facets` that can basically do anything it wants to a `Filter` object; it's not specific to this class.